### PR TITLE
Make curl downloads verbose

### PR DESCRIPTION
**Pull request by kirankt**
Ensure that the curl downloads have the verbose flags set.

### CODE REVIEW

**Review by elfosardo — APPROVED** [review]

File: get-resource.sh [hunk 1]
```diff
@@ -23,25 +23,25 @@ TMPDIR=$(mktemp -d -p /shared/tmp)
 # get header info from the cache
 ls -l
 if [ -n "$CACHEURL" -a ! -e $FFILENAME.headers ] ; then
-    curl -g --verbose --fail -O "$CACHEURL/$FFILENAME.headers" || true
+    curl -v -g --verbose --fail -O "$CACHEURL/$FFILENAME.headers" || true
```
Comment (hardys) [inline get-resource.sh:26]: From `curl --help` I see:

```
-v, --verbose       Make the operation more talkative
```

So we're already setting verbose mode, right?

**Comment by hardys** [comment]
I'm confused, the downloads are already have `--verbose`, what am I missing?

FWIW I was about to propose a PR *removing* `--verbose` because in my environment every download causes thousands of lines like:

```
* TLSv1.3 (IN), TLS app data, [no content] (0):
{ [1 bytes data]

$ sudo podman logs ipa-downloader 2>&1 | grep TLS | wc -l
26412
```

I'm not sure logging *every byte* of the download is super helpful, is it?

**Comment by kirankt** [comment]
> I'm confused, the downloads are already have `--verbose`, what am I missing?
You are correct, Steve. This is an error. I'll close the PR. I created this PR for this upstream repo after the downstream one.

**Pull request closed without merging**